app/Models/Scopes/AuthScope.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models\Scopes;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Scope;
+
+class AuthScope implements Scope
+{
+    /**
+     * Apply the scope to a given Eloquent query builder.
+     */
+    public function apply(Builder $builder, Model $model): void
+    {
+        $builder->where('is_active',true);
```

Comment:
```suggestion
        $builder->where('is_active', true);
```
Sau , luôn có dấu cách e nhé